apps/core0/helper/socat/resolve.go

```diff
@@ -57,8 +57,19 @@ func Resolve(address string) string {
 		return address
 	}
 
+	var (
```

Comment:
@muhamadazmy really not sure about this one to be enough to behave properly

Reply:
I think it should be fine.

Reply:
hmm, giving it another thought, this can match multiple entries now. we should find the one that matches this address